### PR TITLE
feat: Switch Media Device

### PR DESCRIPTION
**Pull request by neel-ds**
### #7 Issue Completed ✅ 
---
This PR adds a feature of setting the available media device. Please check it out @vrajdesai78

### CODE REVIEW

File: src/components/BottomBar/BottomBar.tsx [hunk 1]
```diff
@@ -84,7 +83,7 @@ const BottomBar: React.FC<BottomBarProps> = () => {
 
       {/* Bottom Bar Center */}
       <div className="flex items-center gap-4">
-        {me.role !== 'listener' &&
+        {me.role !== "listener" &&
           (!isAudioOn ? (
             <button
               onClick={() => {
```
Comment (vrajdesai78) [inline src/components/BottomBar/BottomBar.tsx:89]: Here, when calling `fetchAudioStream` you have to pass `audioInputDevice.deviceId`. Do the same for every place wherever `fetchAudioStream` is called. Make sure that whenever passing `deviceId` it's not null.

**Comment by vrajdesai78** [comment]
@neel-ds you have to update Output device in `Audio.tsx` file. You can create a new type as shown below and update it's sink id with `audioOutputDevice.deviceId`. 

Code to declare AudioElement with sinkId

```
type HTMLAudioElementWithSetSinkId = HTMLAudioElement & {
  setSinkId: (id: string) => void;
};
```

**Comment by neel-ds** [comment]
Hey @vrajdesai78, I have fixed the issue raised in the review.

**Comment by vrajdesai78** [comment]
> Hey @vrajdesai78, I have fixed the issue raised in the review.

Great work @neel-ds, LGTM 🚀

**Pull request merged**